ytop-chart/templates/deployment.yaml

```diff
@@ -47,6 +47,10 @@ spec:
           value: {{ quote .Values.controllerManager.manager.env.ytLogLevel }}
         - name: KUBERNETES_CLUSTER_DOMAIN
           value: {{ quote .Values.kubernetesClusterDomain }}
+        {{- if .Values.controllerManager.manager.watchNamespace }}
+        - name: WATCH_NAMESPACE
+          value: {{ .Values.controllerManager.manager.watchNamespace }}
+        {{- end }}
```

Comment:
This file is generated.

Possible environment variables are defined in config/manager/manager.yaml

In trunk watched namespaces are controlled by map DefaultNamaspaces
https://github.com/kubernetes-sigs/controller-runtime/blob/v0.16.5/pkg/cache/cache.go#L188

Where "" is = AllNamespaces
https://github.com/kubernetes-sigs/controller-runtime/blob/v0.16.5/pkg/cache/cache.go#L128